### PR TITLE
ReaperScans: Added Lasted manga

### PR DESCRIPTION
**Pull request by pizidavi**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt [hunk 1]
```diff
@@ -31,25 +31,17 @@ import java.util.Calendar
 class ReaperScans : ParsedHttpSource() {
 
     override val name = "Reaper Scans"
-
```
Comment (Riztard) [inline src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt:34]: dont remove this space and the others. it helps readability

Comment (ArjixWasTaken) [reply src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt:34]: I strongly disagree, how does it help readability?

Comment (alessandrojean) [reply src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt:34]: Reading methods and variables spaced by empty lines is much better and easy to read and find stuff you meed than a lot of grouped ones.

Comment (ArjixWasTaken) [reply src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt:34]: finding stuff? ok, it certainly can help with that
but reading? not really

I agree with spacing certain stuff to make it more readable, but that does not include every single line...

Comment (pizidavi) [reply src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt]: > ...with spacing certain stuff to make it more readable, but that does not include every single line

I agree... in any case, I have put them back

**Pull request merged**